protocol/synthetix/contracts/storage/Pool.sol

```diff
@@ -404,23 +404,25 @@ library Pool {
 
     /**
      * @dev Returns the specified account's collateralization ratio (collateral / debt).
+     * @dev If the account's debt is negative or zero, returns an "infinite" c-ratio.
      */
     function currentAccountCollateralizationRatio(
         Data storage self,
         address collateralType,
         uint128 accountId
     ) internal returns (uint256) {
-        (, uint256 getPositionCollateralValueD18) = currentAccountCollateral(
+        int256 positionDebtD18 = updateAccountDebt(self, collateralType, accountId);
+        if (positionDebtD18 <= 0) {
+            return type(uint256).max;
```

Comment:
this is indeed the best value to pick for 'infinity" because the hex just looks like 0xffffff...